### ISSUE TITLE
Is it possible to detect request cancellation?

### ISSUE THREAD

**Issue by nike61** [msrd0/QtWebApp#31]
Hello, we need a way to detect when clients abort their connection. For example in ASP there is a concept of cancellatkiontoken. Does this library have anything similar?

**Comment by msrd0** [comment]
I don't think this library supports it. Not a 100% sure as I only read most of the code and it's been years since I did.

**Comment by nike61** [comment]
Ok, thanks
